src/java/picard/util/IntervalListScatterer.java

```diff
@@ -33,7 +34,16 @@
          * which is one of the objectives of scattering.</li>
          * </ol>
          */
-        BALANCING_WITHOUT_INTERVAL_SUBDIVISION
+        BALANCING_WITHOUT_INTERVAL_SUBDIVISION,
+        /**
+         * A scatter approach that differs from {@link Mode#BALANCING_WITHOUT_INTERVAL_SUBDIVISION} in a few ways.
```

Comment:
This seems like only one way. :smile:  So the ol probably should go, or it could be fleshed out with more li's describing how this differs.